<?xml version="1.0"?>
<launch>
    <node name="config_node" pkg="ROS_UI_backend" type="configuration.py" output="screen">
    </node>
</launch>  
